packages/Alert/tsconfig.tsbuildinfo

```diff
@@ -1257,7 +1257,7 @@
       },
       "esModuleInterop": true,
       "forceConsistentCasingInFileNames": true,
-      "pathsBasePath": "/Users/ryan/source/byte9/blaze-components-react/packages/build-tools",
+      "pathsBasePath": "/Users/grzegorz/Workspace/JS/blaze-components-react/packages/build-tools",
```

Comment:
Should these files be committed if they contain local user paths as they will change constantly?

[This article](https://betterstack.dev/blog/lerna-typescript-monorepo/#heading-setup-ignore-file) suggests not but not sure if there is a reason to do?

Reply:
We can ignore .tsbuildinfo files as long as we don't want CI to use Typescript incremental builds.

Article above says about one caveat if we delete it though, 

To facilitate this, when TypeScript runs in build mode, a tsconfig.tsbuildinfo file is produced. 
**This file can be safely deleted but it will cause TypeScript to rebuild all dependencies**
That is why the clean command removes this file to create a fresh build.

I will add it go .gitignore for now but we can leave a side-note that if we would like to improve CI times or use Typescript incremental builds, then we should get commit it 👍🏼

Reply:
Incremental builds sounds very useful but does it still work on other developers machines in CI if the paths aren't found? We can keep in and check in CI? Is there are way to set the paths to be relative?

My main worries were about developer specific paths being committed and lots of files changing in each commit and potentially causing conflicts when different developers are working on the repo but I'm happy to keep if works between environments and we can see if the conflicts/files changes are actually an issue.

Reply:
I understand your concern. We can gitignore the build files for now and create a task to check later 👍🏼

Reply:
👍 🚀